src/components/Alerts/Alerts.README.md

```diff
@@ -0,0 +1,80 @@
+# Alerts API
```

Comment:
You can drop the term API and just refer to it as "Alerts" since it is implied that when you are interacting with a library, you are interacting with its API, so it does not need to be mentioned explicitly.